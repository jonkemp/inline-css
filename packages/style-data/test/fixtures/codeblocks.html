<html><link rel="stylesheet" href="codeblocks.css" />
<body>
    {{#if `age <= 40`}}<p>Young</p>{{/if}}</body></html>
    