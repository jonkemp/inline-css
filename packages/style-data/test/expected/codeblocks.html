<html><head><link rel="stylesheet" href="codeblocks.css"></head><body>
    {{#if `age <= 40`}}<p>Young</p>{{/if}}
    </body>
</html>
